iroh/src/commands/sync.rs

```diff
@@ -147,33 +134,80 @@ pub enum Doc {
         /// Optional key prefix (parsed as UTF-8 string)
         prefix: Option<String>,
     },
+    /// Watch for changes and events on a document
+    Watch {
+        /// Document to operate on.
+        ///
+        /// Required unless the document is set through the IROH_DOC environment variable.
+        /// Within the Iroh console, the active document can also set with `doc switch`.
+        #[clap(short, long)]
+        doc_id: Option<NamespaceId>,
```

Comment:
nit: I wonder if just `doc` would be a bit cleaner as argument name

Reply:
Fixed